docs/tutorials-samples/targeting-core-for-new-libraries.md

```diff
@@ -0,0 +1,298 @@
+# Targeting with .NET Core Libraries
+
+Targeting .NET Core for new libraries can be done a number of ways, depending on the platform(s) you wish to support.
+
+## How do I target .NET Core?
+
+First things first:
+
+For the purposes of building a library, targeting ".NET Core" means targetting the .NET Standard Platform.  [This document](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md) outlines everything you need to know, in-depth, about what that means.
+
+A distilled version is this:
+
+You need to pick a version of `dotnetXX` (where `XX` is a version number) to add to your `project.json` file.
+
+1. If you don't care about backwards compatibility, target `dotnet55`.
+2. If you care about compatibility with any of the following:
+
+    - .NET Framework versions from 4.5 and up (and you don't need [these libraries](LINK))
+    - Windows Phone
+    - Windows Phone Silverlight
+    - Universal Windows Platform
+    - DNX Core
+    - Xamarin Platforms
+    - Mono 
+
+    Then [refer to this table](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md#existing-net-standard-platform-versions) and choose the version of the `dotnetXX` moniker you need.
+3. If you care about compatibility with the .NET Framework versions 4.0 or below, or need to support [these .NET 4.5 libraries](LINK), skip to the next section.
+
+For example, if you wanted to have compatibility with .NET Core and .NET Framework 4.6, you would pick `dotnet55`:
+
+```
+{
+    "frameworks":{
+        "dotnet55":{}
+    }
+}
+```
+
+If you wanted to be compatible with Windows Phone Silverlight, you would pick `dotnet51`:
+
+```
+{
+    "frameworks":{
+        "dotnet51":{}
+    }
+}
+```
+
+And that's it!
+
+## How do I target both .NET Core and .NET Framework?
+
+You may need a library to work on .NET Core *and* .NET Framework 4.0 or older.  You may also need to use specific libraries available in .NET 4.5 which are unavailble in .NET Core (such as `SYSTEM.FIXME.SOON`).  For these scenarios, you will need to specifically target the versions of .NET Framework you support.
+
+This is done by adding relevant Target Framework Moniker, such as `net45` to your `project.json` file.  Below is a sample `project.json` file which targets .NET 4.0, .NET 4.5, and .NET Core.
+
+```
+{
+    "dependencies":{
+        "System.Runtime":"4.0.0-rc1-*"
+    },
+    "frameworks":{
+        "net40":{},
+        "net45":{},
+        "dotnet":{}
```

Comment:
Good catch.  This should be "dotnet55" to signify a desire to use the most up-to-date libraries for .NET Core.